examples/pages/react/WithPaginationDocs.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+
+import * as Markup from '../../js/react';
+
+import {WithPagination} from '../../../app-typescript';
+import {range} from 'lodash';
+
+function getItems(pageNo: number): Promise<{items: Array<number>, itemCount: number}> {
+    return new Promise((resolve) => {
+        setTimeout(() => {
+            return resolve({items: range(1, 500).slice((pageNo - 1) * 20, (pageNo - 1) * 20 + 20), itemCount: 500});
+        }, 1000);
+    });
+};
+
+export class WithPaginationDocs extends React.Component {
+    render() {
+        return (
+            <section className='docs-page__container'>
+                <h2 className='docs-page__h2'>With pagination</h2>
+                <p className='docs-page__paragraph'></p>
+                <Markup.ReactMarkup>
+                    <Markup.ReactMarkupPreview>
+                        <WithPagination
+                            getItems={(pageNo) => getItems(pageNo)}
+                        >
+                            {
+                                (items) =>
+                                    <div style={{display: 'flex', flexDirection: 'column', alignItems: 'center'}}>
+                                        {items.map((x, i) => <div key={i}>{x}</div>)}
+                                    </div>
+                            }
+                        </WithPagination>
+                    </Markup.ReactMarkupPreview>
+                    <Markup.ReactMarkupCode>{`
+                        <WithPagination
+                            getItems={(pageNo) => getItems(pageNo)}
+                        >
+                            {
+                                (items) =>
+                                    <div
+                                        style={{
+                                            display: 'flex',
```

Comment:
In docs you'd want to avoid clutter and only show essential usage info. Styles here are that clutter that doesn't add anything to usage information.

Reply:
I thought this should be accurate to represent the actual code of how the given example is implemented. Also I don't think it's clutter as it shows the functionality that you can style the paginated output however you like it.

Reply:
Even if that `div` was without styles - it would still be clear that you can style it, the same way it's clear that you can add other attributes, event listeners etc. even if they're not in the example - if you show that it's possible to use your element you make it clear then that everything is possible that you can do with an element.